params/params_tw.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.89231
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Red Kite TW "
```

Comment:
I think there is an extra space here.

```suggestion
species.name <- " Red Kite TW"
```